smash-optics/src/Data/Can/Optics.hs

```diff
@@ -40,39 +41,39 @@ import Optics.Traversal
 -- ------------------------------------------------------------------- --
 -- Traversals
 
--- | A 'Optics.Traversal' of the first parameter, suitable for use
+-- | An 'AffineTraversal' of the first parameter, suitable for use
```

Comment:
Can you confirm that these haddocks build? It doesn't look like the doc builder for hackage server treated `optics` well.

Reply:
I checked by adding `optics-core` to cabal.project.local and generating haddocks, symbols are linked. `Optics.Traversal` had no chance of working because it's not a symbol (I'd have to be `Optics.Traversal.Traversal`).

I'm not sure why symbols from type signatures are not linked on hackage, seems like a local hackage issue as it works fine e.g. in `aeson-optics`.